src/main/kotlin/org/rust/ide/annotator/RsHighlightingAnnotator.kt

```diff
@@ -120,9 +122,7 @@ private fun partToHighlight(element: RsElement): TextRange? {
     }
 
     if (element is RsMacroCall) {
-        var range = element.referenceNameElement.textRange ?: return null
-        range = range.union(element.excl.textRange)
-        return range
+        return element.excl.textRange
```

Comment:
I split highlighting to 2 annotations because:
1. There is a bug: if multiple tokens are highlighted with a single annotation, this highlighting isn't visible in language injections
2. It's simpler to implement

It there an example of similar highlighting in other languages?